### PR TITLE
Describe on how to very TLS certificates of scrape targets

### PR DESCRIPTION
**Pull request by rmohr**
Document user-facing changes of certificate rotation introduction in https://github.com/kubevirt/kubevirt/pull/2027.

### CODE REVIEW

File: administration/monitoring.adoc [hunk 1]
```diff
@@ -102,6 +102,31 @@ spec:
       insecureSkipVerify: true
 ----
 
+Starting from kubevirt 0.17, the served TLS certificates are signed by the k8s
```
Comment (stu-gott) [inline administration/monitoring.adoc]: How about?

Starting from version 0.17, KubeVirt's TLS certificates are signed...

**Comment by rmohr** [comment]
/hold

@stu-gott thanks for reviewing. https://github.com/kubevirt/kubevirt/pull/2027 is not yet merged, so didn't make it into 0.17.

**Comment by fabiand** [comment]
What is needed to finalize this PR?

**Pull request closed without merging**